collector/benchmarks/ctfe-uninit-stress/src/lib.rs

```diff
@@ -0,0 +1,32 @@
+#![feature(const_fn, const_let)]
```

Comment:
We don't seem to be using `const_let` in the code, maybe we can drop that? Generally we want benchmarks to work on stable as much as possible to avoid needing to patch them over time.